pkg/koordlet/util/cgroup_parent_utils.go

```diff
@@ -49,6 +50,20 @@ func GetKubeQoSByCgroupParent(cgroupDir string) corev1.PodQOSClass {
 	return corev1.PodQOSGuaranteed
 }
 
+// GetPodCgroupDirWithKube gets the full pod cgroup parent dir with the podParentDir (excluding kubepods dir).
+// @podKubeRelativeDir kubepods-burstable.slice/kubepods-pod7712555c_ce62_454a_9e18_9ff0217b8941.slice/
+// @return kubepods.slice/kubepods-burstable.slice/kubepods-pod7712555c_ce62_454a_9e18_9ff0217b8941.slice/
+func GetPodCgroupDirWithKube(podParentDir string) string {
```

Comment:
Please do not use `GetPodCgroupDirWithKube` anymore, as it is just removed recently. Instead, use the `podMeta.podParentDir` directly. See #1129.

Reply:
done